projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -766,16 +779,214 @@ function wpcom_add_new_page_check( $post_id, $post ) {
+		$annotation['content'],
+		function ( $page ) {
+			if ( 'page' !== $page['post_type'] ) {
+				return false;
+			}
+
+			if ( 'about' === $page['post_name'] || false !== strpos( $page['post_title'], 'About' ) ) {
+				return true;
+			}
+		}
+	);
+
+	// Return null if there are no 'About' pages.
+	if ( empty( $headstart_about_pages ) ) {
+		return null;
+	}
+
+	// Get the hs_old_ids for the 'About' pages. We'll use these to find any published pages with the same hs_old_id.
+	$headstart_about_page_hs_old_ids = array_map(
+		function ( $about_page ) {
+			return $about_page['hs_old_id'];
+		},
+		$headstart_about_pages
+	);
+
+	// Return null if there aren't any hs_old_ids to check.
+	if ( empty( $headstart_about_page_hs_old_ids ) ) {
+		return null;
+	}
+
+	$filters     = array(
+		'post_type'    => 'page',
+		'numberposts'  => 1,
+		'fields'       => 'ids',
+		'orderby'      => 'meta_value',
+		'meta_key'     => '_hs_old_id',
+		'meta_compare' => 'IN',
+		'meta_value'   => $headstart_about_page_hs_old_ids,
+	);
+	$about_pages = get_posts( $filters );
+
+	// Return null if we couldn't find any published pages matching the hs_old_ids.
+	if ( empty( $about_pages ) ) {
+		return null;
+	}
+
+	// Return the id of the first About page.
+	return (int) $about_pages[0];
+}
+
+/**
+ * Check to see if a string has been translated.
+ *
+ * @param string $string The string to check.
+ * @param string $domain The text domain to use.
+ * @return bool True if the string has been translated, false otherwise.
+ */
+function wpcom_launchpad_has_translation( $string, $domain = 'jetpack_mu_wpcom' ) {
+	if ( empty( $string ) ) {
+		return false;
+	}
+
+	$current_locale = get_user_locale();
+	if ( is_string( $current_locale ) && 0 === strpos( $current_locale, 'en' ) ) {
+		return true;
+	}
+
+	// phpcs:disable WordPress.WP.I18n.NonSingularStringLiteralText, WordPress.WP.I18n.NonSingularStringLiteralDomain
+	$translated_string = __( $string, $domain );
+
+	return $translated_string !== $string;
+}
+
+/**
+ * Determine `update_about_page` task visibility. The task is visible if there is an 'About' page on the site.
+ *
+ * @return bool True if we should show the task, false otherwise.
+ */
+function wpcom_is_update_about_page_task_visible() {
+	$task_definitions = wpcom_launchpad_get_task_definitions();
+
+	// The task isn't visible if it isn't defined.
+	if ( ! isset( $task_definitions['update_about_page'] ) ) {
+		return false;
+	}
+
+	// The task isn't visible if the get_title callback isn't defined or isn't callable.
+	if ( ! isset( $task_definitions['update_about_page']['get_title'] ) || ! is_callable( $task_definitions['update_about_page']['get_title'] ) ) {
+		return false;
+	}
+
+	// The task isn't visible if we're in a non-English locale and the task title hasn't been translated.
+	$update_about_page_title = $task_definitions['update_about_page']['get_title']();
+	if ( ! wpcom_launchpad_has_translation( $update_about_page_title ) ) {
+		return false;
+	}
```

Comment:
The function in `$task_definitions['update_about_page']['get_title']` is already calling `__()`, so this check isn't 100% correct. Instead, I think we should check using the raw string. Furthermore, I don't think we need to check the task and callback are fully valid, because we don't need to call them.
```suggestion
	// The task isn't visible if the task title hasn't been translated into the current locale.
	if ( ! wpcom_launchpad_has_translation( 'Update your About page', 'jetpack-mu-wpcom' ) ) {
		return false;
	}
```